frontend/utils/zkp.ts

```diff
@@ -0,0 +1,33 @@
+import localforage from "localforage";
+
+const LOAD_URL = "https://d27ahxc61uj811.cloudfront.net/";
```

Comment:
is it insecure to have this name in repo?

Reply:
nah it has to be exposed on the client either way to download the zkey

Reply:
ah ofc